test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -213,6 +216,53 @@ localTest iot nio = do
     assertEqual "expect /local to return gas for tx" (_crGas res) 5
     assertEqual "expect /local to succeed and return 3" e (Right (PLiteral $ LDecimal 3))
 
+localContTest :: IO (Time Micros) -> IO ChainwebNetwork -> TestTree
+localContTest iot nio = testCaseSteps "local continuation test" $ \step -> do
+    cenv <- _getClientEnv <$> nio
+    sid <- mkChainId v maxBound (0 :: Int)
+    mv <- newMVar 0
+
+    step "execute /send with initial pact continuation tx"
+    batch <- firstStep mv
+    rks <- liftIO $ sending sid cenv batch
+
+    step "check /poll responses to extract pact id for continuation"
+    PollResponses m <- polling sid cenv rks ExpectPactResult
+    pid <- case NEL.toList (_rkRequestKeys rks) of
+      [rk] -> case HashMap.lookup rk m of
+        Nothing -> assertFailure "impossible"
+        Just cr -> case _crContinuation cr of
+          Nothing -> assertFailure "not a continuation - impossible"
+          Just pe -> return $ _pePactId pe
+      _ -> assertFailure "continuation did not succeed"
+
+    step "execute /local continuation dry run"
+    SubmitBatch batch2 <- secondStep mv pid
+    r <- _pactResult . _crResult <$> local sid cenv (head $ toList batch2)
+    case r of
+      Left err -> assertFailure (show err)
+      Right (PLiteral (LDecimal a)) | a == 2 -> return ()
+      Right p -> assertFailure $ "unexpected cont return value: " ++ show p
+  where
+    tc =
+      "(namespace 'free)(module m G (defcap G () true) (defpact p () (step (yield { \"a\" : (+ 1 1) })) (step (resume { \"a\" := a } a))))(free.m.p)"
+
+    ttl = 2 * 24 * 60 * 60
+    pm = Pact.PublicMeta (Pact.ChainId "0") "sender00" 100_000 0.01 ttl
+
+    firstStep mv = modifyMVar mv $ \ !n -> do
+      t <- toTxCreationTime <$> iot
+      kps <- testKeyPairs sender00 Nothing
+      c <- Pact.mkExec tc A.Null (pm t) kps (Just "fastTimedCPM-peterson") Nothing
+      return (succ n, SubmitBatch $ pure c)
+
+    secondStep mv pid = do
+      nonce <- (<>) "nonce" . sshow @Natural <$> readMVar mv
+      t <- toTxCreationTime <$> iot
+      kps <- testKeyPairs sender00 Nothing
+      c <- Pact.mkCont pid 1 False A.Null (pm t) kps (Just nonce) Nothing (Just "fastTimedCPM-peterson")
```

Comment:
For here, same as above but instead of `mkExec'` it's 
```
... $ mkCont $ mkContMsg pid 1
```

Reply:
Done!